### PR TITLE
standard header

### PR DESCRIPTION
**Pull request by dalekunce**

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -1,46 +1,101 @@
...
-            <img src={ missingMapsLogo } width="94px" alt="Missing Maps logo "/>
+            <img
+              src="http://www.missingmaps.org/assets/graphics/meta/MM-White.svg"
+              width="94px"
+              alt="Missing Maps logo "
+            />
           </a>
         </div>
-        <div className="nav-list">
+        <div class="nav-list">
```
Comment (mojodna) [inline src/components/Header.js]: This needs to be `className` for React.

File: src/components/Header.js [hunk 2]
```diff
@@ -1,46 +1,101 @@
...
           <ul>
-            <a href="http://missingmaps.org/contribute/"><li className="nav-item">CONTRIBUTE</li></a>
-            <li className="nav-item nav-dropdown">EVENTS
-              <div className="dropdown-content nav-events">
-                <a href="http://missingmaps.org/events/"><div className="nav-item">CURRENT EVENTS</div></a>
-                <a href="http://missingmaps.org/host"><div className="nav-item">ORGANIZE AN EVENT</div></a>
+            <li class="nav-item nav-dropdown">
```
Comment (mojodna) [inline src/components/Header.js]: `className`

Comment (mojodna) [reply src/components/Header.js]: (There are lots of these that need to be fixed.)

**Comment by dalekunce** [comment]
@mojodna does that work?

**Comment by mojodna** [comment]
👍 Looks good to me.

**Pull request merged**